src/glue/resources/dataset_crawler_assignments.yaml

```diff
@@ -3,6 +3,10 @@ standard:
   - sharedSchema_v1
   - WeatherResult_v1
   - sharedSchema_for_Gradual_Onset_v1
+  - 3DRotation_v1
```

Comment:
Are we informed when new datasets are added by our collaborators or bridge?

Reply:
Short answer: yes. AFAIK nothing's been formalized (whatever that would look like), but everyone involved recognizes that BD needs to be part of the QA process. Sonia has been our link between the assessment developers (Northwestern, in this case) and ourselves so far.